lib/matplotlib/cbook.py

```diff
@@ -527,11 +545,7 @@ def process(self, s, *args, **kwargs):
         """
         if s in self.callbacks:
             for cid, proxy in list(six.iteritems(self.callbacks[s])):
-                # Clean out dead references
-                if proxy.inst is not None and proxy.inst() is None:
-                    del self.callbacks[s][cid]
-                else:
-                    proxy(*args, **kwargs)
+                proxy(*args, **kwargs)
```

Comment:
Err, that should have read

```python
try:
    proxy(*args, **kwargs)
except ReferenceException:
    self.remove_proxy(proxy)
```
Anything involving an event loop makes me twitchy.  The cost of the `try` is minimal and may prevent inscrutable impossible to reproduce errors later.  In this case I would rather be safe than sorry.